src/Esy.re

```diff
@@ -1,3 +1,4 @@
+/** Esy API **/
```

Comment:
There are still a couple of promise rejections in this file, is this intentional?